Nancy.sln.DotSettings

```diff
@@ -33,6 +33,8 @@
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/STICK_COMMENT/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/CodeStyle/CSharpUsing/AddImportsToDeepestScope/@EntryValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/CodeStyle/CSharpUsing/QualifiedUsingAtNestedScope/@EntryValue">True</s:Boolean>
+	<s:String x:Key="/Default/CodeStyle/FileHeader/FileHeaderRegionName/@EntryValue"></s:String>
+	<s:String x:Key="/Default/CodeStyle/FileHeader/FileHeaderText/@EntryValue"></s:String>
```

Comment:
The commit message for c90d342e334e93848750c714c5fe6ae5f5595f83 explains it quite well, I think?

> Explicitly set the ReSharper file header stuff to blank so global config is not used instead.

I have ReSharper with a standard file header that is added when I do cleanup since Nancy does not explicitly set the header to blank. Since Nancy has no headers, I think it make sense setting this to nothing, agree?